src/data/api.ts

```diff
@@ -149,3 +149,63 @@ export async function fetchArticles(
     return null;
   });
 }
+
+export async function fetchLearningpathMeta(
+  learningPathId: string,
+  context: Context,
+): Promise<GQLResourceMeta> {
+  const response = await fetch(
+    `/learningpath-api/v2/learningpaths/${learningPathId}`,
+    context,
+  );
+  const learningPath = await resolveJson(response);
+  return {
+    id: learningPath.id,
+    title: learningPath.title.title,
+    introduction: learningPath.introduction
+      ? learningPath.introduction.introduction
+      : undefined,
+    metaDescription: learningPath.description
+      ? learningPath.description.description
+      : undefined,
+    metaImage: learningPath.coverPhoto
+      ? learningPath.coverPhoto.url
+      : undefined,
+    lastUpdated: learningPath.lastUpdated,
+  };
+}
+
+export async function fetchArticleMeta(
+  articleId: string,
+  context: Context,
+): Promise<GQLResourceMeta> {
+  const response = await fetch(
```

Comment:
Don't call article-converter here, use article-api if you don't need the article content. Should probably use articlesLoader in the resolvers if possible.